json_text.js

```diff
@@ -257,22 +258,22 @@ arrPopups = [
             "в атмосферному повітрі населених місць (Зареєстровано в Міністерстві юстиції України 10 лютого 2020 р. за № 156/34439). " +
             "<a href=\"https://zakon.rada.gov.ua/laws/show/z0156-20#Text\" target=\"_blank\">https://zakon.rada.gov.ua/laws/show/z0156-20#Text</a>. ",
         "text_mn": "<div class=\"i-header\">Хязгаарлалт</div>" +
-            "<b>Хууль эрх зүйн үндэслэл болон Удирдамжийг хэн боловсруулдаг вэ?</b>" +
+            "<b>Эрх зүйн зохицуулалт, стандартыг хэн хариуцан боловсруулдаг юм бол?</b>" +
             "<br><br>" +
-            "Дэлхийн эрүүл мэндийн байгууллагаас (ДЭМБ) баталсан <a href=\"https://apps.who.int/iris/handle/10665/345329\" target=\"_blank\">Агаарын Чанарын Удирдамж</a>" +
-            "ид агаар бохирдуулагч бодисуудын агууламжийг (эрүүл мэндэд нөлөөгүй эсвэл бага нөлөөтэй) тодорхойлсон байдаг. Эдгээр утгыг шинжлэх ухааны нотолгоонд үндэслэн " +
-            "тодорхойлсон бөгөөд бодлого боловсруулагчдад агаарын чанарын стандартыг тогтоохдоо баримтлах үндэслэл болгоход зориулагдсан болно. Агаарын бохирдлын урт хугацааны " +
-            "үр нөлөөний хувьд  ихэнх бохирдуулагчдад нөлөөллийн босго агууламж байдаггүй хэмээн өнөөгийн шинжлэх ухааны  нотолгоонд тулгуурлан судлаачид таамаглаж байна. " +
-            "\"Нөлөөллийн босго\" гэдэг нь эрүүл мэндэд хортой нөлөө үзүүлэхгүй байгаа бохирдуулагчийн агууламжийг хэлнэ. ДЭМБ одоогоор агаарын чанарын удирдамжаа шинэчилж байна." +
+            "Дэлхийн эрүүл мэндийн байгууллагаас (ДЭМБ) баталсан <a href=\"https://apps.who.int/iris/handle/10665/345329\" target=\"_blank\">Агаарын чанар тодорхойлох удирдамж</a>" +
+            "ид агаар бохирдуулагч бодисын агууламжийг эрүүл мэндэд үзүүлэх нөлөөгөөр нь тодорхойлсон байдаг. Эдгээр утгыг шинжлэх ухааны судалгаа нотолгоонд үндэслэн тодорхойлсон " +
+            "бөгөөд бодлого боловсруулагчдад агаарын чанарын стандарт тогтооход ашиглах зөвлөмж юм. Агаарын бохирдлын урт хугацааны үр нөлөөний хувьд ихэнх бохирдуулагчийн " +
+            "сөрөг нөлөөг хэмжих босго утга байхгүй гэж өнөөгийн шинжлэх ухааны нотолгоонд тулгуурлан судлаачид таамаглаж байна. \"Нөлөөллийн босго\" гэдэг нь эрүүл мэндэд " +
+            "хортой сөрөг нөлөө үзүүлэхгүй хэмжээнд байгаа бохирдуулагчийн агууламжийг хэлнэ. ДЭМБ одоогоор агаарын чанарын удирдамжаа шинэчилж байна." +

```

Comment:
267 when I translate this with google translate it says that the WHO is currently reviewing the Air quality guidelines. However, they published them in 2021. Could you please check if this is the correct translation?

Reply:
@Nara1469 See as reference the English text [here](https://github.com/SwissTPH/AP-Health-Effects/blob/2df07ecb3419b4ea37f36fa8082394695e2e8c2d/json_text.js#L146-L160).

Reply:
Thank you for the update, @Nara1469